src/jit/lsraxarch.cpp

```diff
@@ -2437,6 +2437,21 @@ void LinearScan::TreeNodeInfoInitHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
         info->srcCount += GetOperandSourceCount(intrinsicTree->gtOp.gtOp2);
     }
     info->srcCount += GetOperandSourceCount(intrinsicTree->gtOp.gtOp1);
+
+#ifdef _TARGET_X86_
+    if (intrinsicTree->gtHWIntrinsicId == NI_SSE42_Crc32)
```

Comment:
Is this specific to CRC32 op2? Do we need a more general mechanism here to specify if op1 or op2 required byte registers on x86?

Reply:
> Is this specific to CRC32 op2? Do we need a more general mechanism here to specify if op1 or op2 required byte registers on x86?

I believe that only CRC32 will use "byte".